web/docs/language/basic-elements.md

```diff
@@ -365,7 +369,33 @@ export const signUp = async (args, context) => {
 ```
 
 :::info
-You don't need to worry about hashing the password yourself! Even when you are using Prisma's client directly and calling `create()` with a plain-text password, Wasp put middleware in place that takes care of hashing it before storing it to the database.
+You don't need to worry about hashing the password yourself! Even when you are using Prisma's client directly and calling `create()` with a plain-text password, Wasp put middleware in place that takes care of hashing it before storing it to the database. An additional middleware also performs field validation.
+:::
+
+##### Customizing user entity validations
+
+To disable default validations, or add your own, you can do:
+```js
+const newUser = context.entities.User.create({
```

Comment:
It might be helpful to provide a more "realistic" example here: e.g. we leave `_waspSkipDefaultValidations` to be `false` (and comment it can be omitted or set explicitly to true) and for custom validations we use something like what you did in a feature proposal, e.g. "password must contain at least one digit". Just a thought - it took me a bit to realize that we turned off default validations and then re-written them as custom ones.

Reply:
ah sorry, I see now that "one digit" is also part of default validations. Maybe something like "uppercase letter".

Reply:
Good thinking on the variety and more realistic. I will do should contain an uppercase letter, thanks!

Reply:
This has been addressed here: https://github.com/wasp-lang/wasp/pull/376/commits/9a545b74ddf9be9e69eff94f13d72b19886f2acc Thanks!